.github/workflows/build_and_test.yml

```diff
@@ -51,7 +51,7 @@ on:
 jobs:
   precondition:
     name: Check changes
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
In Spark 3.5, I think we should use ubuntu-22.04

Reply:
Okay, made a minimize change.